packages/settings-library/src/settings_library/r_clone.py

```diff
@@ -15,3 +15,17 @@ class S3Provider(str, Enum):
 class RCloneSettings(BaseCustomSettings):
     R_CLONE_S3: S3Settings = Field(auto_default_from_env=True)
     R_CLONE_PROVIDER: S3Provider
+
+    # SEE https://rclone.org/docs/#transfers-n
+    R_CLONE_OPTION_TRANSFERS: NonNegativeInt = Field(
+        5, description="`--transfers X`: sets the amount of parallel transfers"
+    )
+    # SEE https://rclone.org/docs/#retries-int
+    R_CLONE_OPTION_RETRIES: NonNegativeInt = Field(
+        3, description="`--retries X`: times to retry each individual transfer"
+    )
+    # SEE https://rclone.org/docs/#buffer-size-size
+    R_CLONE_OPTION_BUFFER_SIZE: str = Field(
+        "0M",
```

Comment:
MINOR: can this be done using https://docs.pydantic.dev/latest/api/types/#pydantic.types.ByteSize instead of `str`??

Reply:
I'm unsure `rclone` uses this format. I will leave it as is